src/style.css

```diff
@@ -2,2 +1,72 @@
+html {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+}
+
+header {
+  width: 100%;
+  background-color: #f8f8f8;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 20px;
+  height: 60px;
+}
+
+.logo img {
+  height: 40px;
+  display: block;
+}
+
+nav ul {
+  display: flex;
+  list-style: none;
+}
+
+nav ul li {
+  margin-right: 20px;
+}
+
+nav ul li:last-child {
+  margin-right: 0;
+}
+
+nav ul li:first-child {
+  margin-left: 0;
+}
+
+nav a {
+  text-decoration: none;
+  color: #333;
+  display: block;
+  text-align: center;
+  line-height: 60px;
+}
+
+nav a:hover {
+  color: #007bff;
+}
+
+is-active {
+  color: #007bff;
+  position: relative;
+}
+
+is-active::after {
```

Comment:
The selector `is-active::after` should be `.is-active::after` to correctly apply the pseudo-element to the class in HTML.